src/main/java/org/opentripplanner/openstreetmap/tagmapping/DefaultMapper.java

```diff
@@ -502,8 +502,17 @@ public void populateProperties(WayPropertySet props) {
 
     // We assume highway/cycleway of a cycle network to be safer (for bicycle network relations, their network is copied to way in postLoad)
     // this uses a OR since you don't want to apply the safety multiplier more than once.
+    // Cyclestreets where bicycles have priority or are the only permitted vehicles (a thing in traffic rules of
+    // some european countries) get an identical safety multiplier. See e.g. https://nl.wikipedia.org/wiki/Fietsstraat
+    // For simplicity these two concepts are handled together.
```

Comment:
I discussed this with our team's OSM guy and we thought it might not make sense to give same safety value to streets where there can be cars (even if bicycles would have a priority) as to normal cycle network cycleways.

Reply:
In Germany at least all of these are shared between bikes and cars.

Reply:
rcn/lcn/ncn doesn't say anything about whether cars are allowed. Often they are - it's just quieter roads.

Reply:
I agree that bike exclusive ways should be given a higher safety value then shared ways. But from my understanding none of the cycle network tags signal exclusive use for bikes. https://wiki.openstreetmap.org/wiki/Cycle_routes

Whether or not the way is shared is encoded in the highway tag https://wiki.openstreetmap.org/wiki/Bicycle#Cycle_features

with highway=path or highway=cycleway when its not for motorized vehicles.
https://github.com/opentripplanner/OpenTripPlanner/blob/dev-2.x/src/main/java/org/opentripplanner/openstreetmap/tagmapping/DefaultMapper.java#L74